src/test/java/core/basesyntax/ReportServiceImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static Storage storage;
+    private static StringBuilder validReport;
+    private static StringBuilder validReportEmptyFile;
+
+    @BeforeClass
+    public static void setUp() {
+        reportService = new ReportServiceImpl();
+        storage = new Storage();
+        storage.storage.put("banana", 25);
+        storage.storage.put("apple", 25);
+        validReport = new StringBuilder();
+        validReport.append("fruit,quantity");
+        validReport.append(System.lineSeparator());
+        validReport.append("banana,25");
+        validReport.append(System.lineSeparator());
+        validReport.append("apple,25");
+        validReport.append(System.lineSeparator());
+        validReportEmptyFile = new StringBuilder();
+        validReportEmptyFile.append("fruit,quantity");
+        validReportEmptyFile.append(System.lineSeparator());
+    }
+
+    @Test
+    public void generateReportEmptyStorage_Ok() {
+        storage.storage.clear();
```

Comment:
it's better to add a method to clear the storage after each test or after all , depending on the test case